### PR TITLE
Add team section component

### PR DESCRIPTION
**Pull request by EmCeeEs**
Fix #92 

*Remark*: I did not add a component to the homepage because I lack the data (like pictures, etc.). Also we might wanna add the section to the homepage once we have brought back the cms.

### CODE REVIEW

**Comment by Levino** [comment]
What's the WIP?

**Comment by EmCeeEs** [comment]
Could not manage to align the social icons.
image

File: package.json [hunk 1]
```diff
@@ -90,11 +91,12 @@
     "eslint-plugin-react": "^7.16.0",
     "eslint-plugin-react-hooks": "^1.7.0",
     "eslint-plugin-standard": "^3.0.1",
+    "faker": "^4.1.0",
```
Comment (Levino) [inline package.json]: No need, right?

File: package.json [hunk 2]
```diff
@@ -90,11 +91,12 @@
...
     "gatsby-plugin-typescript": "^2.1.15",
     "prettier": "^1.18.2",
     "ramda": "^0.26.1",
     "react-docgen-typescript-loader": "^3.3.0",
     "reactstrap": "^8.1.1",
     "typescript": "^3.6.4"
   }
-}
+}
```
Comment (Levino) [inline package.json]: Newline please.

File: src/components/Team/index.tsx [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+import * as R from 'ramda'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faGithub, faLinkedinIn } from '@fortawesome/free-brands-svg-icons'
+import { IconProp } from '@fortawesome/fontawesome-svg-core'
+
+export enum Platforms {
+  github = 'github',
+  linkedIn = 'linkedIn',
+}
+export type Platform = Platforms
```
Comment (Levino) [inline src/components/Team/index.tsx]: Whats this for?

File: src/components/Team/index.tsx [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+  <a
+    className="list-inline-item px-1"
+    href={url}
+    title={platform}
+    target="_blank"
+    rel="noopener noreferrer"
+  >
+    <FontAwesomeIcon icon={SocialMediaIcons[platform]} size="lg" />
+  </a>
+)
```
Comment (Levino) [inline src/components/Team/index.tsx]: Please move this whole "social icon" stuff in its own file.

File: src/components/Team/index.tsx (hunk 3 above)
Comment (EmCeeEs) [reply src/components/Team/index.tsx]: Just a type alias for the enum object. Can be removed.

**Pull request merged**